things/postgres/things.go

```diff
@@ -73,7 +73,21 @@ func (tr thingRepository) Save(ctx context.Context, ths ...things.Thing) ([]thin
 }
 
 func (tr thingRepository) Update(ctx context.Context, t things.Thing) error {
-	q := `UPDATE things SET name = :name, metadata = :metadata WHERE id = :id;`
+	var query []string
+	var upq string
+	if t.Name != "" {
```

Comment:
Can we update with empty name or empty metadata anymore?

Reply:
No

Reply:
This means that if the name or the metadata has been edited there is no way to revert it. Is this what we want?

Reply:
Let me revert